third_party/move/move-compiler-v2/tests/checking/typing/vector_mismatched_args.exp

```diff
@@ -0,0 +1,31 @@
+
+Diagnostics:
+error: `integer` not compatible with `bool` in vector expression
```

Comment:
This error message could be improved, but it's not urgent.  Ideally you'd collect the various parameter types and mention them all at once:

error: vector arguments not compatible: `integer` not compatible with `bool`

Reply:
Outdated, this time widely sometime yesterday.

Reply:
Yep.